wire-schema/src/jvmTest/kotlin/com/squareup/wire/schema/DirectedAcyclicGraphTest.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2025 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire.schema
+
+import assertk.assertThat
+import assertk.assertions.hasMessage
+import assertk.assertions.isEqualTo
+import assertk.assertions.isLessThan
+import kotlin.test.Test
+import kotlin.test.fail
+
+class DirectedAcyclicGraphTest {
+
+  @Test
+  fun singleNode() {
+    val nodes = listOf("A")
+    val graph = DirectedAcyclicGraph(nodes) { emptyList() }
+    val order = graph.topologicalOrder()
+    assertThat(listOf("A")).isEqualTo(order)
```

Comment:
```suggestion
    assertThat(order).isEqualTo(listOf("A")
```
You can also do
```suggestion
    assertThat(order).containsExactly(listOf("A")
```
but have to import it